functions.php

```diff
@@ -0,0 +1,36 @@
+<?php
+require "connectToDb.php";
+
+/*
+ * function getArtworks
+ * returns an array of the artworks
+ */
+
+function getArtworks() : array
+{
+    $db = getDB();
+    $query = $db->prepare('SELECT * FROM `artworks`;');
+    $query->execute();
+    return $query->fetchAll();
+}
+
+/*
+ * function displayArtworks
+ * @params array
+ * returns HTML displaying the artwork details
+ */
+
+function displayArtworks(array $data) : string
+{
+    $return = '';
+    foreach ($data as $artwork){
```

Comment:
What if there are no items in the DB? Some kind of error message here would be a good touch!